cloud/azure/modules/app/main.tf

```diff
@@ -158,6 +169,22 @@ resource "azurerm_app_service" "civiform_app" {
     vnet_route_all_enabled               = true
   }
 
+  # We will only mount this storage container if SAML authentication is being used
+  dynamic "storage_account" {
```

Comment:
should this be azurerm storage account or is it different?

Reply:
it's different. this needs to have the same name as the block it's generating, which is this https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service#storage_account

Reply:
ahh it's within the app_service, my b! makes sense